avatars/avatars.js

```diff
@@ -1649,16 +1651,16 @@ class Avatar {
           if (aIndex !== -1) {
             morphTargetInfluences[aIndex] = volumeValue;
           }
-          if (eIndex !== -1) {
+          if (eIndex !== -1 && !this.manuallySetMouth) {
```

Comment:
I think this is not correct? We are already preventing `.volume` update (https://github.com/webaverse/app/pull/2835/files/7ff71e32fb96c6dc5e2fe598f47d4101047202dc#diff-fcb2093ac11d9b04a179d3dedbef5c0aae4d14b7f71db90e483becca4170b7ccR1922) so why do we also prevent the update here?

Reply:
Because if I don't ban the 'e', 'i', 'o', 'u' index, it would change the mouth figure:

https://user-images.githubusercontent.com/60634884/164544866-93743cfe-dc47-4806-ba92-569c1042aa63.mp4



https://user-images.githubusercontent.com/60634884/164544918-937a9c79-36e3-41e5-a3a4-4c13027f66d6.mp4


https://user-images.githubusercontent.com/60634884/164544931-08cbdf1b-1923-4127-bb27-86c36ab94229.mp4

Reply:
Then it sounds like instead of banning it, the ADSR code/player binding should be setting the correct vowels instead.

E.g. it can simple enable only the `A` and set the rest to `0`.

Reply:
move setting vowels in player-avatar-binding https://github.com/webaverse/app/pull/2835/commits/5f4304a25d46c7a70698592c3473b949942639bb